app/models/billable_occurrence.rb

```diff
@@ -6,6 +6,12 @@ class BillableOccurrence < UuidApplicationRecord
   belongs_to :billable, polymorphic: true
   has_many :billable_occurrence_rate_types, dependent: :destroy
   has_many :rate_types, through: :billable_occurrence_rate_types
+
+  accepts_nested_attributes_for :billable, allow_destroy: true
+
+  def build_billable(params)
```

Comment:
This is the start of an implementation I was looking into here: https://stackoverflow.com/questions/3969025/accepts-nested-attributes-for-with-belongs-to-polymorphic/32915379 but I'm not really sure what they're suggesting you do for the controller; most of the examples I've found are for form inputs, and while they do map to attributes sometimes, it's a little tricky